README.md

```diff
@@ -481,29 +481,11 @@ Install to the workbench's virtualenv by running the following command form the
 pip install -r requirements.txt
 ```
 
-In the main XBlock repository, create the following configuration file
-in `workbench/settings_mentoring.py` in the XBlock repository:
-
-```python
-from settings import *
-
-INSTALLED_APPS += ('mentoring',)
-DATABASES['default']['NAME'] = 'workbench.sqlite'
-```
-
-Because this XBlock uses a Django model, you need to sync the database
-before starting the workbench. Run this from the XBlock repository
-root:
-
-```bash
-$ ./manage.py syncdb --settings=workbench.settings_mentoring
```

Comment:
@e-kolpakov No, it's no longer required because I removed all the Django models from this repository, so it no longer needs to be added to `INSTALLED_APPS`. Instead, it uses XBlock setuptools entry points to register with workbench. So all that matters is that your installation of xblock-sdk is working and you have installed this xblock into the same virtual environment - it no longer needs any other configuration.